Source/StarRunner2019/HallwayActor.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2019 Dulce Studios. All Rights Reserved.
+
+#include "HallwayActor.h"
+
+#include "HallwayJointComponent.h"
+#include "HallwayUnitComponent.h"
+
+#include "StarRunner2019Character.h"
+
+#include "Kismet/KismetMathLibrary.h"
+#include "Engine/Classes/Kismet/KismetSystemLibrary.h"
+
+#include <stdlib.h> // for rand function LOL
```

Comment:
Should probably use <cstdlib> instead.  So sayeth the gods of stack overflow lmao https://stackoverflow.com/a/2900792